vllmd-hypervisor/generate-runtime-image.sh

```diff
@@ -0,0 +1,593 @@
+
+#!/usr/bin/env bash
+#
+# generate-runtime-image.sh - Create a runtime image for VLLMD Hypervisor
+#
+# This script generates a Debian-based runtime image by downloading a Debian netinst ISO,
+# creating a preseed configuration disk, and using cloud-hypervisor-v44 to generate a 
+# runtime image with the preseed configuration. The resulting image can be used with VLLMD Hypervisor.
+#
+# Usage:
+#   bash generate-runtime-image.sh [OPTIONS]
+#
+# Options:
+#   --dry-run              Show what would be done without making any changes
+#   --force                Force overwrite of existing files
+#   --state-dir=PATH       Set custom state directory (default: $HOME/.local/state/vllmd/vllmd-hypervisor)
+#   --output=PATH          Set custom output path for the runtime image
+#   --memory=SIZE          Memory size for generation process in GiB (default: 16GiB)
+#   --disk-size=SIZE       Size of the output disk image in GiB (default: 20GiB)
+#   --debian-version=VER   Debian version to use (default: bookworm)
+#   --preseed=PATH         Path to custom preseed file (default: use built-in preseed-v1-bookworm.cfg)
+#
+# Examples:
+#   bash generate-runtime-image.sh
+#   bash generate-runtime-image.sh --dry-run
+#   bash generate-runtime-image.sh --force --memory=32GiB --disk-size=40GiB
+#   bash generate-runtime-image.sh --preseed=/path/to/custom-preseed.cfg
+#
+
+set -euo pipefail
+
+# Process command line arguments
+DRY_RUN=0
+FORCE=0
+STATE_DIR="${HOME}/.local/state/vllmd/vllmd-hypervisor"
```

Comment:
Use the style `PURPOSE_PATH` for any directory without a file, and `PURPOSE_FILEPATH` for any file (which should include a fully quaified path). Make this change throguhout the pr